week-3/Homework/mandatory/1-oxygen-levels.js

```diff
@@ -7,12 +7,41 @@ Their computer detects a list of nearby planets that have Oxygen in their atmosp
 To be safe, they need to land on the first unamed planet that has Oxygen levels between 19.5% and 23.5%.
 
 Write a function that finds the oxygen level of the first safe planet - Oxygen between 19.5% and 23.5%
+
+Some string methods that might help you here are .replace() and .substring(). Let's look at a quick
+example before trying the exercise.
 */
 
-function safeLevels() {
+/* .replace() allows us to add something where we removed something*/
+let greeting = "Good Morning";
+greeting.replace('Morning', 'Evening'); // outputs Good Evening
+
+
+/* .substring() allows us to remove things from strings */
+let dessert = "ice cream and pancakes";
+
+let newdessert = dessert.substring(0, 9);
+
+console.log(newdessert); // returns ice cream
 
+/* + + + + + + + */
+/* Now try the exercise */
+function removePercentagerSign(arr){
+var newArr=arr.map(key=>key.replace('%',''));
+return newArr;
 }
 
+
+console.log(removePercentagerSign(["24.2%", "11.3%", "19.9%", "23.1%", "29.3%", "20.2%"]));
+
+function safeLevels(newArr1) {
+  var castedArray=removePercentagerSign(newArr1);
```

Comment:
```suggestion
  var castedArray = removePercentagerSign(newArr1);
```
Remember to leave spaces between your `=` operator and your variable name. Good name for this function `removePercentageSign` 😄